backend/src/errorHandler.ts

```diff
@@ -0,0 +1,28 @@
+import { Logger } from "pino";
+
+import logger from "./logger";
+
+class ErrorHandler {
+    private readonly logger: Logger;
+
+    constructor({ logger }: { logger: Logger }) {
+        this.logger = logger;
+    }
+
+    public async handleExceptionError(error: Error) {
+        this.logger.error(`Uncaught exception: ${error.message}`);
+        // TODO: add monitoring
+        process.exit(1);
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    public async handleRejectionError(reason: any) {
+        this.logger.error(reason, "Unhandled rejection: ");
```

Comment:
Did you intend to have it like this?

Reply:
yes, this will print:
```
[1636711720145] ERROR: Unhandled rejection: 
    Error: error message
        at /Users/apple/Documents/subspace-relayer/backend/dist/index.js:131:15
        at Object.<anonymous> (/Users/apple/Documents/subspace-relayer/backend/dist/index.js:134:3)
        at Module._compile (node:internal/modules/cjs/loader:1101:14)
        at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
        at Module.load (node:internal/modules/cjs/loader:981:32)
        at Function.Module._load (node:internal/modules/cjs/loader:822:12)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
        at node:internal/main/run_main_module:17:47
        ```

Reply:
Why is `reason` before `Unhandled rejection:`?

Reply:
first param is object, second is message
https://getpino.io/#/docs/api?id=loggererrormergingobject-message-interpolationvalues